### PR TITLE
(feat)Add some missing translations to the cretaepatientform

### PR DESCRIPTION
**Pull request by jona42-ui**
# Objective

Add traslations to the createpatient form

# Issue
https://github.com/orgs/I-TECH-UW/projects/6/views/1?pane=issue&itemId=43998079

##  Screens
# Before
[Screencast from 11-15-2023 06:04:46 PM.webm](https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/78595738/170a56f2-24ff-42af-96b8-b2dd4ef78de5)

# After
[Screencast from 11-16-2023 08:22:42 AM.webm](https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/78595738/5cb4854c-9db1-428a-bc52-e44f3fc430f7)

### CODE REVIEW

**Comment by jona42-ui** [comment]
@mozzy11 
updated PR here

File: frontend/src/components/patient/CreatePatientForm.js [hunk 1]
```diff
@@ -366,84 +371,97 @@ function CreatePatientForm(props) {
...
-                          undefined
-                            ? ""
-                            : values.patientContact?.person.primaryPhone
-                        }
-                        name={field.name}
-                        labelText={`Contact Phone: ${configurationProperties.PHONE_FORMAT}`}
-                        id={field.name}
-                        className="inputText"
-                      />
+              <AccordionItem title={intl.formatMessage({ id: "emergencyContactInfo.title" })}>
```
Comment (mozzy11) [inline frontend/src/components/patient/CreatePatientForm.js:384]: Thanks @jona42-ui 
Can you still get rid of the formatting changes here ?? 
its even wrong indentation

**Comment by mozzy11** [comment]
Thanks @jona42-ui . This looks better but you still have some formating changes some where . 
can you also close the old PR ??

File: frontend/src/components/patient/CreatePatientForm.js (hunk 1 above)
Comment (jona42-ui) [reply frontend/src/components/patient/CreatePatientForm.js:384]: okay on it!

**Comment by jona42-ui** [comment]
okay on it! thanks

**Comment by mozzy11** [comment]
Great , Looks clean now

**Comment by jona42-ui** [comment]
@mozzy11  i will picking up another component unless you have any caveats!

**Pull request merged**